test/verify/check-journal

```diff
@@ -222,5 +222,56 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
                          [ "check-journal", "BEFORE BOOT", "" ]
                      ])
 
+    def testBinary(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute(
+"""
+python -c '
+import socket
+s = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+s.connect("/run/systemd/journal/socket")
+s.send("MESSAGE=Hello \\01 World\\nPRIORITY=3\\nFOO=bar\\nBIN=a\\01b\\02c\\03\\n")'
+""")
+
+        self.login_and_go("/system/logs")
+
+        sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('[13 bytes of unprintable data]')"
+        b.wait_present(sel)
+        b.wait_visible(sel)
+        b.click(sel)
+
+        b.wait_text("#journal-entry-message", "[13 bytes of unprintable data]")
+
+        def wait_field(name, value):
+            row_sel = "#journal-entry-fields tr:contains('" + name + "')"
+            b.wait_present(row_sel + " td:nth-child(2):contains('" + value + "')")
+
+        wait_field("FOO", "bar")
+        wait_field("BIN", "[6 bytes of unprintable data]")
+
+    def testNoMessage(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute(
+"""
+python -c '
+import socket
+s = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+s.connect("/run/systemd/journal/socket")
+s.send("PRIORITY=3\\nFOO=bar\\n")'
```

Comment:
> I agree, although this will bring us close to quoting hell...

I was thinking that just `"PRIORITY=3\\nFOO=bar\\n"` could be the parameter string, as is. This doesn't have to be foolproof or add another layer, just reduce copy&paste a bit.